src/create-client.ts

```diff
@@ -1,10 +1,10 @@
 import { IProxyConfig } from './config';
 import Client from './client';
-import { initialize } from 'unleash-client';
+import { initialize, Unleash } from 'unleash-client';
 import Metrics from 'unleash-client/lib/metrics';
 import { defaultStrategies } from 'unleash-client/lib/strategy';
 
-export const createClient = (config: IProxyConfig): Client => {
+export const createSingletonClient = (config: IProxyConfig): Client => {
```

Comment:
Because the createClient is an exported should we keep it? Maybe it can be an alias to `createSingletonClient` and it can be marked as deprecated.

Reply:
createClient is only exported internally and used into the app.ts. We don't export it to proxy users. I created this create-client.ts a week ago to encapsulate dependency injection graph creation.